### PR TITLE
Exporting processed food (Exporting PR 2/?)

### PR DESCRIPTION
**Pull request by ZombiesVsPlants**
## About The Pull Request

Makes it so you can sell the foods (ingredients) that you make at a processor

## Why It's Good For The Game

Cook can now sell some food and maybe cargo will listen to cook a bit more

## Hotel?
trivago

## Changelog
:cl:
add: Selling processed foods to centcomm
/:cl:

### CODE REVIEW

**Comment by ZombiesVsPlants** [comment]
Bruh Trovis why?

**Comment by TheFakeElon** [comment]
Could buff the price a little, can’t wait to see chef burger some cargo techs for stealing his food

**Comment by ZombiesVsPlants** [comment]
Prices pretty much doubled the price, better or too OP?

**Comment by TheFakeElon** [comment]
Seems good!

**Comment by MCterra10** [comment]
changelog malformed.
I'll re-execute travis.

File: austation/code/modules/cargo/exports/food_and_drink/processed.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  
+     /datum/export/cornchips
+	cost = 30
+	unit_name = "cornchip"
+	export_types = list(/obj/item/reagent_containers/food/snacks/cornchips)
+  
+     /datum/export/roastparsnip
+	cost = 20
+	unit_name = "roast parsnip"
+	export_types = list(/obj/item/reagent_containers/food/snacks/roastparsnip)
```
Comment (MCterra10) [inline austation/code/modules/cargo/exports/food_and_drink/processed.dm:54]: all your shit has the wrong indentation.

Comment (ZombiesVsPlants) [reply austation/code/modules/cargo/exports/food_and_drink/processed.dm:54]: Ahhhhhh fuck

Comment (ZombiesVsPlants) [reply austation/code/modules/cargo/exports/food_and_drink/processed.dm]: done

**Pull request merged**